test/micro/org/openjdk/bench/java/lang/StackWalkBench.java

```diff
@@ -52,11 +53,11 @@ public class StackWalkBench {
     private static final StackWalker WALKER =
             StackWalker.getInstance(RETAIN_CLASS_REFERENCE);
     private static final StackWalker WALKER_CLASS_ONLY =
-            StackWalker.getInstance(DROP_METHOD_INFO, RETAIN_CLASS_REFERENCE);
+            StackWalker.getInstance(Set.of(DROP_METHOD_INFO, RETAIN_CLASS_REFERENCE));
 
     static StackWalker walker(String name) {
         return switch (name) {
-            case "default" -> WALKER;
+            case "class+method" -> WALKER;
```

Comment:
don't you need to also change "default" into "class+method" in the `@Param({"default", "class_only"})` annotations below?

Reply:
thanks for catching it. will fix.